packages/muze/src/canvas/components/message-component.js

```diff
@@ -0,0 +1,41 @@
+import { selectElement, makeElement, applyStyle } from 'muze-utils';
+import MuzeComponent from './muze-chart-component';
+
+export default class MessageComponent extends MuzeComponent {
+    constructor (params) {
+        super(params.name, params.config.dimensions, 0);
+        this.setParams(params);
+    }
+
+    render (container) {
+        const parent = selectElement(container);
+        const { className } = this.params.config.classPrefix;
```

Comment:
`this.params.config` is being references multiple times. store it in a variable or use destructuring.